qemu/tests/timerdevice_clock_drift_with_sleep.py

```diff
@@ -36,11 +36,11 @@ def verify_elapsed_time():
         get_time_cmd += ' taskset -c $i sh /tmp/sleep.sh; done'
         guest_cpu = session.cmd_output("grep 'processor' "
```

Comment:
Let's use this param `guest_cpu` in the for blocker in L34

Reply:
Updated.
(1/1) Host_RHEL.m8.u0.raw.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.myprovider.timerdevice.clock_drift_with_sleep: PASS (1208.91 s)